### PR TITLE
PBL-32223 Move default templates to SDK

### PR DESCRIPTION
**Pull request by cjwilliams**
@Katharine

### CODE REVIEW

File: pebble_tool/commands/sdk/__init__.py [hunk 1]
```diff
@@ -5,11 +5,8 @@
 import subprocess
 import logging
 
-from pebble_tool.exceptions import (ToolError, MissingSDK, PebbleProjectException, InvalidJSONException,
-                                    InvalidProjectException, OutdatedProjectException)
+from pebble_tool.exceptions import (ToolError, MissingSDK)
```
Comment (Katharine) [inline pebble_tool/commands/sdk/__init__.py]: Omit parens if not spanning multiple lines.

File: pebble_tool/commands/sdk/create.py [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
 import errno
+import json
 import os
 import uuid
 
-from ..base import BaseCommand
+from shutil import copy2
+
+from . import SDKCommand
+from pebble_tool.sdk import SDK_VERSION, sdk_version
```
Comment (Katharine) [inline pebble_tool/commands/sdk/create.py]: I regret my naming choices.

File: pebble_tool/commands/sdk/create.py [hunk 3]
```diff
@@ -35,40 +37,57 @@ def __call__(self, args):
...
+            default_main = os.path.join(project_template_path, 'simple.c')
+        else:
+            default_main = os.path.join(project_template_path, 'main.c')
+        copy2(default_main, os.path.join(project_src, "{}.c".format(project_name)))
 
         # Add appinfo.json file
-        appinfo_dummy = DICT_DUMMY_APPINFO.copy()
-        appinfo_dummy['uuid'] = str(uuid.uuid4())
-        appinfo_dummy['project_name'] = project_name
+        with open(os.path.join(project_template_path, 'appinfo.json')) as f:
```
Comment (Katharine) [inline pebble_tool/commands/sdk/create.py]: This is a template string. In order to maintain the and formatting of the template, it'd probably be best to use that and just perform string substitution.

File: pebble_tool/commands/sdk/create.py (hunk 2 above)
Comment (cjwilliams) [inline pebble_tool/commands/sdk/create.py]: `SDK_TEMPLATE_VERSION`?

File: pebble_tool/commands/sdk/create.py [hunk 4]
```diff
@@ -1,17 +1,20 @@
 from __future__ import absolute_import, print_function
 
 import errno
+import json
```
Comment (Katharine) [inline pebble_tool/commands/sdk/create.py]: Is this actually used?

**Comment by Katharine** [comment]
This looks pretty reasonable to me, apparently-stray import aside. :+1:

File: pebble_tool/commands/sdk/create.py (hunk 4 above)
Comment (cjwilliams) [inline pebble_tool/commands/sdk/create.py]: Nope, forgot to remove that.

**Pull request merged**